app/models/campaign.rb

```diff
@@ -42,7 +43,7 @@ class Campaign < ActiveRecord::Base
   # Scopes #
   ##########
 
-  scope :active, -> { where('end > ?', Time.zone.now) }
+  scope :active, -> { where('end > ? OR (start IS NULL AND end IS NULL)', Time.zone.now) }
```

Comment:
Having just one NULL would cause an error, currently https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/models/campaign.rb#L120